aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -140,7 +142,9 @@ struct cublasCommonArgs {
       Tensor& c,
       const std::optional<Tensor>& scale_a = std::nullopt,
       const std::optional<Tensor>& scale_b = std::nullopt,
-      const std::optional<Tensor>& scale_result = std::nullopt) {
+      const std::optional<Tensor>& scale_result = std::nullopt,
+      const std::optional<ScalingType>& scaling_choice_a = std::nullopt,
+      const std::optional<ScalingType>& scaling_choice_b = std::nullopt) {
```

Comment:
For block wise scale, the scale_a/b should be a tiled tensor, suggest to update the comments of this function 
https://github.com/pytorch/pytorch/blob/6f59bfa14276df34a239f1474620b4a5a03746ab/aten/src/ATen/native/cuda/Blas.cpp#L1211-L1212